march_hardware_interface/src/march_hardware_interface_node.cpp

```diff
@@ -48,8 +48,7 @@ int main(int argc, char** argv)
   }
 
   controller_manager::ControllerManager controller_manager(&march, nh);
-  const ros::Duration minimal_elapsed_time = ros::Duration(march.getEthercatCycleTime() / 1000.0);
-  ros::Time last_update_time = ros::Time::now() - minimal_elapsed_time;
+  ros::Time last_update_time = ros::Time::now()
```

Comment:
```suggestion
  ros::Time last_update_time = ros::Time::now();
```
;) Also nice commit name

Reply:
oh shit